lib/assets/constants.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+const EdgeInsetsGeometry screenPadding = EdgeInsets.all(
+  32.0,
+);
+const int startDate = 1975;
+const int endDate = 2029;
+const int nameMinLength = 3;
+const SizedBox sizedBoxFormPage = SizedBox(
+  height: 32.0,
+);
+const SizedBox sizedBoxHeightEight = SizedBox(
+  height: 8.0,
+);
+const SizedBox sizedBoxHeightFour = SizedBox(
+  height: 4.0,
+);
+const SizedBox sizedBoxResponsePage = SizedBox(
+  height: 16.0,
+);
+const double appBarElevation = 0.0;
+const double alertDialogBorderRadius = 8.0;
+const double dividerThickness = 2.0;
+const String appBarTitle = "Form";
+const Color elevatedButtonEnabledColor = Colors.indigoAccent;
+Color elevatedButtonDisabledColor = Colors.grey.shade300;
```

Comment:
> Why is this not a constant?

Colors.grey.shade300 cannot be a constant because it cannot be fully determined at compile-time. The Colors.grey is a named constant, but the shade300 property is not a constant, but rather a getter method that computes a value at runtime based on the current color space.

Reply:
> Please add color constants in a different class.

Done.